account_fiscal_position_rule/models/account_invoice.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
Remove This line [https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration)